Emby.Naming/AudioBook/AudioBookNameParserResult.cs

```diff
@@ -14,5 +16,50 @@ public struct AudioBookNameParserResult
         /// Gets or sets optional year of release.
         /// </summary>
         public int? Year { get; set; }
+
+        /// <summary>
+        /// Comparison equallity operators for <see cref="AudioBookNameParserResult"/>.
```

Comment:
```suggestion
        /// Equality operator for <see cref="AudioBookNameParserResult"/>.
```

Same of the other occurrences